src/main/java/mate/academy/dao/BookDaoImpl.java

```diff
@@ -30,7 +30,7 @@ public Book create(Book book) {
             statement.setObject(INDEX_OF_SECOND_PARAM, book.getPrice());
 
             int affectedRows = statement.executeUpdate();
-            if (affectedRows < 1) {
+            if (affectedRows == 0) {
```

Comment:
Not resolved. Vasyl's comment 'It is okay to affect zero rows' means you do not need to throw the exception here. The same is in second method. The only place where affected rows matter is in delete method where you return boolean value based on the value of affected rows.